cinch/roles/jenkins_master/templates/etc/nginx/conf.d/jenkins_https.conf

```diff
@@ -38,7 +43,13 @@ server {
     proxy_set_header        X-Forwarded-Proto $scheme;
     proxy_redirect http:// https://;
     proxy_pass              http://jenkins;
-  }
+    {% if jenkins_nginx_rate_limiting == "true" %}
+    limit_req zone=jenkins burst={{ nginx_burst_queue }} nodelay;
+    limit_conn addr  {{ nginx_max_connections_per_ip }};
+    limit_rate_after {{ nginx_bandwidth_limit_after_size }};
+    limit_rate       {{ nginx_max_bandwidth_outbound }};
+    {% endif %}
+	}
```

Comment:
Is the indentation level here correct?

Reply:
Thanks - should be fixed now if you can double check me